test/libsolidity/Imports.cpp

```diff
@@ -104,8 +104,7 @@ BOOST_AUTO_TEST_CASE(simple_alias)
 BOOST_AUTO_TEST_CASE(library_name_clash)
 {
 	CompilerStack c;
-	c.addSource("a", "library A {} pragma solidity >=0.0;");
-	c.addSource("b", "library A {} pragma solidity >=0.0;");
+	c.addSource("a", "library A {} library A {} pragma solidity >=0.0;");
```

Comment:
After this change, the test does not test the intended feature anymore. There is a separate module that checks for library name clashes. This might be something we need to address here, too: Clashes in the linker.